src/Sulu/Bundle/WebsiteBundle/Resolver/ParameterResolver.php

```diff
@@ -85,6 +85,10 @@ public function resolve(
             $structureData = [];
         }
 
+        if (!$requestAnalyzer) {
+            return [];
```

Comment:
I dont know if its better throw a LogicException here if requestAnalyzer does not exists

Reply:
not sure if there is a valid usecase where this is called without a `$requestAnalyzer` 🙈 
think i would just return the `$parameter` array in this case to be safe